docs.html

```diff
@@ -283,12 +285,71 @@ <h2 class="component-title">Input</h2>
                         rows="3"></textarea>
                 </div>
                 <iframe class="component-code"
```

Comment:
Try using `embed` html tag.

Reply:
Got to know this new HTML. I will research more into this and let iframe be the solution for now.

Reply:
Sure